finagle-thrift/src/main/scala/com/twitter/finagle/thrift/ThriftClientBufferedCodec.scala

```diff
@@ -26,7 +26,15 @@ object ThriftClientBufferedCodec {
   /**
    * Create a [[com.twitter.finagle.thrift.ThriftClientBufferedCodecFactory]]
    */
-  def apply() = new ThriftClientBufferedCodecFactory
+  def apply(): ThriftClientBufferedCodecFactory =
+    apply(Protocols.binaryFactory(), true)
+
+  def apply(protocolFactory: TProtocolFactory): ThriftClientBufferedCodecFactory =
+    apply(protocolFactory, true)
+
+  def apply(protocolFactory: TProtocolFactory,
```

Comment:
My intent with the changes to `ThriftClient(Buffered|Framed)Codec(Factory)` was to provide a way for client code that was still using the builder interface with `.codec(...)` to make use of the new option. I'm more than happy to remove all of them if you guys are ok limiting control of the new feature to `with`-methods / Stack API - that seems to be the case - can you confirm?